bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -109,15 +102,14 @@ class App extends Component {
     this.setState({ enableResize: shouldEnableResize });
   }
 
-  renderPoll() {
-    const { pollIsOpen } = this.props;
-
-    if (!pollIsOpen) return null;
+  renderPanel() {
+    const { enableResize } = this.state;
 
     return (
-      <div className={styles.poll}>
-        <PollContainer />
-      </div>
+      <PanelManager
+        enableResize={enableResize}
+        openPanel={Session.get('openPanel')}
```

Comment:
This needs to be inside a tracker otherwise wont work properly. You never should use any reactive source on the component render itself.